snapshots/wb/2023-08-14/pip_api.py

```diff
@@ -0,0 +1,1093 @@
+import io
+import time
+from multiprocessing.pool import ThreadPool
+from pathlib import Path
+
+import click
+import numpy as np
+import pandas as pd
+import requests
+from botocore.exceptions import ClientError
+from joblib import Memory
```

Comment:
@Marigold By the way, we need to add joblib to `pyproject.toml`

Let me know all the things I need to set up after you have your tests ready!

Reply:
Oh, right, thanks! Added that. There's nothing else you have to setup, just run `poetry install` and then you should be able to run the script with `python snapshots/wb/2023-08-14/pip_api.py`.